BankWallet/BankWallet/Modules/Restore/RestoreModule.swift

```diff
@@ -1,11 +1,10 @@
 protocol IRestoreView: class {
+    func set(predefinedAccountTypes: [IPredefinedAccountType])
```

Comment:
Maybe it would be better to have ViewItem instead of passing heavy object to view